<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageprocessing</class>
 <widget class="QWidget" name="imageprocessing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Processing</string>
  </property>
  <widget class="QGroupBox" name="OutputImageBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Output Images</string>
   </property>
   <widget class="QListWidget" name="OutputImageList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="TargetDirectoryBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>291</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Target Directory</string>
   </property>
   <widget class="QLabel" name="TargetDirLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:white; border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BrowseTargetDir">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="InputImageBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>291</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Input Images</string>
   </property>
   <widget class="QPushButton" name="ClearImageFiles">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectImageFiles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Image Files</string>
    </property>
   </widget>
   <widget class="QListWidget" name="InputList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>51</y>
      <width>271</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="TerminalWindowBox">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>260</y>
     <width>261</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Terminal Window</string>
   </property>
   <widget class="QTextBrowser" name="TerminalOut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ProgressBarBox">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>0</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Progress Bar</string>
   </property>
   <widget class="QProgressBar" name="encodeProgress">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ImagePreviewBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>0</y>
     <width>691</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Encoded Image Preview</string>
   </property>
   <widget class="QLabel" name="ImageDisplay">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>671</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="EncodingTypeBox">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>60</y>
     <width>261</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Select Encoding Type</string>
   </property>
   <widget class="QPushButton" name="EncodeTest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>1-bit prints 
(1-bit -&gt; 24-bit)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GreyscaleEncode">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>8-bit prints 
(8-bit -&gt; 24-bit)</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>190</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Pixel Binning</string>
   </property>
   <widget class="QComboBox" name="BinSelect">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>21</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1x1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2x2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3x3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4x4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5x5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6x6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>7x7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8x8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9x9</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10x10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11x11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12x12</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>13x13</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>14x14</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>15x15</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>16x16</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>17x17</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>18x18</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19x19</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>20x20</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="BinStart">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Binning</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
